docs/docs/user/references/cli-commands.md

```diff
@@ -179,4 +179,5 @@ $ kpops schema [OPTIONS] SCOPE:{pipeline|config} [COMPONENTS_MODULE]
 
 **Options**:
 
+* `--include-stock-components / --no-include-stock-components`: Include the built-in KPOps components.  [default: include-stock-components]
```

Comment:
why have this as an option, like what's the use case for excluding the builtin components?

Reply:
tbh testing is my reason to add it, but perhaps someone might want to only have a schema for that if they are using only custom components

Do you think I should remove it? I couldn't figure out a better way to abstract from the implementation of the stock components when testing the function.

I hate that I constantly have to generate new snapshots and I would argue that it makes it easier for errors to slip through.

Reply:
ah ok, I see what you mean. tbf I wouldn't say it has value to regular users, but I agree with the testability argument